### DISCUSSION TITLE
How to access root element within a v-scope object method?

### DISCUSSION THREAD

**Discussion by ggedde** [vuejs/petite-vue#2] [Q&A]
Since $refs does not exist and we are suppose to use selectors, I was hoping that I could do `this.$el.querySelector('form')` but this does not work.

I would prefer to not use a custom ID or Class as this is a reusable scope that I am using on multiple sections.
Previously with Alpine I was just using `this.$refs.form`.

I am not using mount and instead just using a sharable object across multiple v-scope elements and using defer and init on the cdn script.

So what is the best way to achieve this?
Thanks

**Comment by yyx990803** [comment]
`$refs` are now supported in 0.2.2.

**Reply by ggedde** [reply]
Awesome, I was just about to mention that I think $refs￼ should be added if there's no way to access the parent element within the scope object.￼

With that said, is there a way to access the parent element from the scope object?

With $refs I don't think it's necessary but just curious.￼￼￼

Thanks
